interface/themes/style_purple.css

```diff
@@ -629,31 +630,38 @@ font-weight: bold;
 }
 
 .css_button_small span {
-    background: transparent url( 'theme_assets/bg_button_span_small.gif' ) no-repeat;
+    /* background: transparent url( 'theme_assets/bg_button_span_small.gif' ) no-repeat; */
```

Comment:
Is there a reason why you didn't just remove this?

Reply:
No reason. OK I will remove it now